src/google/cloud/happybase/table.py

```diff
@@ -142,20 +142,23 @@ def families(self):
 
     def regions(self):
```

Comment:
The signature should match the original happybase implementation.  Please see here: https://github.com/wbolster/happybase/blob/master/happybase/table.py#L76